src/Sarif.Driver/DumpEventsCommand.cs

```diff
@@ -297,8 +297,9 @@ public int Run(DumpEventsOptions options)
                         case "ManifestData":
                         {
                             return;
-
                         }
+
+                        case "EventTrace/PartitionInfoExtension":
```

Comment:
>PartitionInfoExtension [](http://example.com/codeflow?start=41&length=22)

well, okey-doke. :) you collected the legacy form, it appears.

Reply:
If it's obvious what I did to trigger legacy form mode instead of V2, I'd love to be corrected.  This is the command I'm using.  [1]

[1] Sarif.PatternMatcher.cli.exe analyze ..\\..\\..\\..\\..\\..\\AzureDevOps\ --recurse true --max-file-size-in-kb 999999999 --output .\ado.sarif --level "Error;Warning;Note" --plugin ".\SEC101.SecurePlaintextSecretsInternal.json" --log ForceOverwrite --insert ContextRegionSnippets --etw .\ADO-Run.etl